src/pages/Bestellung.js

```diff
@@ -0,0 +1,48 @@
+import styled from "styled-components";
+export default function Bestellung({ localStorage, inputData, totalPrice }) {
+  console.log(localStorage);
+  return (
+    <>
+      <Section>
+        <h4>Prüfe deine daten und bestätige den Kauf</h4>
+        <main>
+          <h4>Dein persönliches Data: </h4>
+          <h5>Name: {inputData.name}</h5>
+          <h5>Email: {inputData.email}</h5>
+          <h5>Adresse: {inputData.address}</h5>
+          <h4>
+            Deie bestellte Artikel:
+            {localStorage.map((element) => (
+              <ul key={element.id}>
+                <div>
+                  {" "}
+                  <h5> Artikel: {element.title}</h5>{" "}
+                  <h5>
+                    {element.amount} x {element.price} €
+                  </h5>
+                </div>
+              </ul>
```

Comment:
An `ul` element needs a `li` element as a child element. Not a div. This is not semantically.

Reply:
already done!